### PR TITLE
Svelte frontend

### PR DESCRIPTION
**Pull request by albin02forsberg**

### CODE REVIEW

File: frontend/src/main.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import './app.css'
+import App from './App.svelte'
+
+const app = new App({
+  target: document.getElementById('app'),
+})
```
Comment (albin02forsberg) [inline frontend/src/main.js:6]: This  is wrong.

**Review by albin02forsberg — COMMENTED** [review]
LGTM!

File: frontend/src/main.js (hunk 1 above)
Comment (albin02forsberg) [reply frontend/src/main.js:6]: Fixed!

**Review by GabrielAdward — APPROVED** [review]
LGTM

**Pull request merged**